recipes-ni/base-files-nilrt/files/sysctl.conf

```diff
@@ -1,12 +1,17 @@
 # Set overcommit_memory to never overcommit
 vm.overcommit_memory=2
 
-# Under overcommit_memory=2, restrict the overcommit_ratio to 95% of
-# CommitLimit, to avoid the OOM killer under "normal" workloads (i.e. which lack
-# unusual device configurations that may consume more kernel memory). This value
-# was obtained totally empirically; values above 98 are known to be unsafe, and
-# 95 is chosen merely as a rounder and more conservative number.
-vm.overcommit_ratio=95
+# Restrict virtual memory allocations to a percentage of the RAM that the
+# kernel makes available for virtual memory (the "CommitLimit" figure in
+# /proc/meminfo).  This setting was determined empirically.
+#
+# The maximum "safe" value of vm.overcommit_ratio (i.e., to avoid the OOM
+# killer) depends on the working set of the system, particularly for those
+# processes which are not mlockall()'d. Values above 98 are known to be unsafe.
+#
+# Note: The overcommit ratio is now set by the nisetcommitratio script using
+# memory reservation amounts defined in ni-rt.ini
```

Comment:
NIT: Could you strike "now" from this comment. The reader probably doesn't care that this value *used to* be set here.